trestle/core/commands/merge.py

```diff
@@ -0,0 +1,24 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Trestle Merge Command."""
+
+from ilcli import Command
+
+
+class MergeCmd(Command):
```

Comment:
A quick question do we want to have all of these commands in their own module or not? it seems very 'java'y and unnecessary. Happy to discus.

Reply:
The purpose of having each command in their own module is to increase code cohesion. It feels to me that having all commands in the same module would inevitably lead to low cohesion.